src/main/base/store.ts

```diff
@@ -323,43 +359,4 @@ export class Store {
             Store.cfg.store = store
         })
     }
-    
-    
-    static pushToCloud(): void {
-        if (Store.cfg.get('connectUser.auth') === null) return;
-        var syncData = Object();
-        if (Store.cfg.get('connectUser.sync.themes')) {
-            syncData.push({
-                themes: Store.cfg.store.themes
-            })
-        }
-        if (Store.cfg.get('connectUser.sync.plugins')) {
-            syncData.push({
-                plugins: Store.cfg.store.plugins
-            })
-        }
-    
-        if (Store.cfg.get('connectUser.sync.settings')) {
-            syncData.push({
-                general: Store.cfg.get('general'),
-                home: Store.cfg.get('home'),
-                libraryPrefs: Store.cfg.get('libraryPrefs'),
-                advanced: Store.cfg.get('advanced'),
-            })
-        }
-        let postBody = {
-            id: Store.cfg.get('connectUser.id'),
-            app: electron.app.getName(),
-            version: electron.app.isPackaged ? electron.app.getVersion() : 'dev',
-            syncData: syncData
-        }
-
-        fetch('https://connect.cidercollective.dev/api/v1/setttings/set', {
```

Comment:
"settings" with 3 t's

Reply:
Thats not my doing, and the api page setup for it also has 3 t's... @quacksire